8.0/Dockerfile

```diff
@@ -0,0 +1,80 @@
+FROM quay.io/centos7/s2i-base-centos7
```

Comment:
We should not add a Dockerfile for RHSCL, which do not have php 8.0.

Reply:
Addressed by https://github.com/sclorg/s2i-php-container/pull/347/commits/b2bfb6c3f2fe144f39784203075ac243a0216f72